ytfzf

```diff
@@ -938,7 +938,7 @@ update_ytfzf () {
 	updatefile="/tmp/ytfzf-update"
 	curl -L "https://raw.githubusercontent.com/pystardust/ytfzf/$branch/ytfzf" -o "$updatefile"
 
-	if sed -n '1p' < "$updatefile" | grep -q '#!/bin/sh' ; then
+	if [ -n "$(cat "$updatefile")" ];  then
```

Comment:
If there was a curl error or gh servers are down, the we get a 404 response. 
so the file won't be empty, it will have some error statement. Thats why I was checking for the shebang

Reply:
ok